nephos/composer/install.py

```diff
@@ -186,7 +187,6 @@ def setup_card(opts, msp_path, user_name, network, roles, verbose=False):
         hlc_cli_ex.execute(
             (
                 "composer card create "
-                + "-n {network} "
```

Comment:
Rather than entirely removing the network, I would propose setting it to "None" by default, and only adding the line if we explicitly specify the network.

Thus, the function signature would change to:

```
setup_card(opts, msp_path, user_name, roles, network=None, verbose=False)
```

And the line 189 would change to:

```
+ "-n {network} " if network else ""
```

Reply:
Otherwise we get rid of the flexibility we added to specify network explicitly, which disallows installing a card to a specific user-created network (e.g. a QueryAdmin, etc.)